epoch/api/channels_api_usage.md

```diff
@@ -1030,6 +1038,116 @@ are the ones considered latest from the channel's perspective. For example the
 next correct off-chain update/deposit/withdrawal shall have a withdraw
 transaction's `round` plus one.
 
+### Getting state
+At any moment in time any participant can ask one's own FSM for various views of
+the latest channel state. This is to help wallet apps but they shall not trust
+FSM and compute state locally.
+
+#### Get balances
+In order to get the balances as those are part from the channel's state tree, a
+participant sends a WebSocket message
+```
+{'action': 'get',
+ 'tag': 'balances'
+ 'payload': {
+    'accounts': ['ak$Y1NRjHuoc3CGMYMvCmdHSBpJsMDR6Ra2t5zjhRcbtMeXXLpLH',
+                 'ak$V6an1xhec1xVaAhLuak7QoEbi6t7w5hEtYWp9bMKaJ19i6A9E']
+  }
+}
+```
+
+The `accounts` section of the payload contains a list of addresses to fetch
+balances of. Those can be either account balances or a contract ones, encoded
+as an account addresses.
+
+A response of this call looks like
+```
+{'action': 'get',
+ 'tag': 'balances'
+ 'payload': [
+		{'account': 'ak$Y1NRjHuoc3CGMYMvCmdHSBpJsMDR6Ra2t5zjhRcbtMeXXLpLH',
+     'balance': 700},
+    {'account': 'ak$V6an1xhec1xVaAhLuak7QoEbi6t7w5hEtYWp9bMKaJ19i6A9E',
+     'balance': 400}
+	]
+}
+```
+
+If a certain account address had not being found in the state tree - it is simply
+skipped in the response.
+
+#### Get proof of inclusion
+In order to build and use different services, one might need to provide a third party
+a minimified view of the internal channel's state.
```

Comment:
Although "minimified" does appear on the web, I'm not sure that it's a proper word. How about "minimal"?